modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/js/components/attributes/__tests__/Attributes.spec.tsx

```diff
@@ -107,10 +107,10 @@ describe('Attributes', () => {
 
 		expect(screen.getByText('products')).toBeInTheDocument();
 
-		expect(screen.getByText('order')).toBeInTheDocument();
+		expect(screen.getByText('order[buy]')).toBeInTheDocument();
```

Comment:
@brianwulbern I am not familiar with Front-end code, but I think the '**order**' here is not the key for language.
and this is the way how Front-end code uses language key.
`Liferay.Language.get('attribute[noun]')`

Reply:
@ling-alan-huang ok, yeah, I'm trying out a way where I clear out that one, and the ones in `Attributes.tsx`